packages/commonwealth/server/router.ts

```diff
@@ -714,19 +720,33 @@ function setupRouter(
   // NOTE: if a successfulRedirect url is used in the options then there is no way to access that data.
 
   router.get('/auth/github', (req, res, next) => {
-    passport.authenticate('github', <any>{state: {hostname: req.hostname}})(req, res, next);
-  })
-  router.get('/auth/github/callback', passport.authenticate('github', {
-    failureRedirect: '/',
-  }), startOAuthLogin.bind(this, models, 'github'))
+    passport.authenticate('github', <any>{ state: { hostname: req.hostname } })(
+      req,
+      res,
+      next
+    );
+  });
+  router.get(
+    '/auth/github/callback',
+    passport.authenticate('github', {
+      failureRedirect: '/',
+    }),
+    startOAuthLogin.bind(this, models, 'github')
+  );
 
   router.get('/auth/discord', (req, res, next) => {
-    passport.authenticate('discord', <any>{state: {hostname: req.hostname}})(req, res, next);
+    passport.authenticate('discord', <any>{
+      state: { hostname: req.hostname },
+    })(req, res, next);
   });
 
-  router.get('/auth/discord/callback', passport.authenticate('discord', {
-    failureRedirect: '/',
-  }), startOAuthLogin.bind(this, models, 'discord'));
+  router.get(
```

Comment:
While this stuff is all fine, in the future can we not include auto-linting in commits?